pyproject.toml

```diff
@@ -22,14 +22,14 @@ classifiers = [
     "Topic :: Utilities",
 ]
 dependencies = [
-    "docutils",
-    "importlib_metadata",
-    "jinja2",
-    "packaging",
-    "pygments",
-    "pyyaml",
-    "reportlab",
-    "smartypants",
+    "docutils~=0.20.0",
+    "importlib-metadata~=8.0",
+    "jinja2~=3.0",
+    "packaging~=24.0",
+    "pygments~=2.0",
+    "pyyaml~=6.0",
+    "reportlab~=4.0",
+    "smartypants~=2.0",
```

Comment:
Placed some limitations on these versions to avoid breaking changes.

For example, "pyyaml~=6.0" means install any version in the 6.x series, but not 7.x or above which could have API breaking changes. Syntactic sugar for `"pyyaml>=6.0,<7".